docs/plugins.rst

```diff
@@ -10,7 +10,8 @@ one may want to perform. Hence developers can extend or even replace
 the GHC standard Plugins with custom implementations.
 
 Two Plugin types exist that can be extended: the `Probe` and `Check` class.
-In v0.7.0 also plugins for Resource Authentication, `ResourceAuth`, were added.
+In v0.7.0 also plugins for Resource Authentication, `ResourceAuth`, were added
+and in v0.8.0 the geocoder plugin was introduced.
```

Comment:
will hopefully be 0.8.1

Reply:
Made a reference to 0.9 with commit 4fa7da7